Documentation/block.md

```diff
@@ -43,9 +43,11 @@ parameters:
 
 Create the storage class.
 ```bash
-kubectl create -f storageclass.yaml
+kubectl create -f replicated-storageclass.yaml
 ```
 
+An advanced example using an erasure coded storage pool with Rook is at the bottom of the page [Advanced Example: Erasure Coded Block Storage](#advanced-example-erasure-coded-block-storage).
```

Comment:
I think this sentence isn't needed. When they scroll down they will see the heading that will seems obvious.